RELEASE/scripts/autoscend/auto_quest_level_13.ash

```diff
@@ -496,23 +705,12 @@ boolean L13_towerNSHedge()
 
 boolean L13_sorceressDoor()
 {
-	if(internalQuestStatus("questL13Final") < 5 || internalQuestStatus("questL13Final") > 5)
+	if(internalQuestStatus("questL13Final") != 5)
```

Comment:
`.to_int()`

Reply:
`internalQuestStatus()` returns an int. This is correct.